implementors.md

```diff
@@ -195,7 +195,7 @@ ACR Artifact Documentation: [aka.ms/acr/artifacts](https://aka.ms/acr/artifacts)
 
   ```sh
   oras push myregistry.azurecr.io/samples/artifact:1.0 \
-      --manifest-config /dev/null:application/vnd.unknown.config.v1+json \
+      --manifest-config /dev/null:application/vnd.unknown.config.v1 \
       ./artifact.txt:application/vnd.unknown.layer.v1+txt
```

Comment:
Based on our call, the layer should be `application/txt`, however I'm not seeing an existing entry. 
However, if we change the defaults, below, the above examples would be:
```sh
 oras push myregistry.azurecr.io/samples/artifact:1.0 \
      --manifest-config /dev/null \
      ./artifact.txt
```
as the defaults within ORAS would set these to:
`application/unknown.config`
and
`application/txt`

Reply:
I had this blocked for requesting review, but I don't want to block you cutting a build tonight. So, digest the feedback, make a call, and cut the release. The main thing I want to change in this release is having new artifacts default to an OCI image. Only container images, that can be run with docker and other container runtimes should use `application/vnd.oci.image.config.v1`